qfdmo/models/acteur.py

```diff
@@ -435,6 +435,11 @@ def sort_key(a):
 
         return orjson.dumps(acteur_dict).decode("utf-8")
 
+    def display_postal_address(self):
+        return bool(
+            self.adresse or self.adresse_complement or self.code_postal or self.ville
+        )
```

Comment:
```suggestion
        return self.adresse or self.adresse_complement or self.code_postal or self.ville
```

Faudrait tester maintenant qu'on a un test unitaire, mais je crois que le `bool` est facultatif ici

Reply:
il faut changer les tests car je teste si `is True` ou `is False`

Mais d'un autre coté, je me dit que c'est pas plus mal que ça retourne un booléen vu le nom de la fonction
Qu'en pense-tu ?

Reply:
J'ai ajouté le type de retour pourque ce soit clair

Reply:
Ha oui, je viens de capter, ça peut valoir none si aucun n'est défini ? 

Dans ma tête c'était automatiquement casté en tant que booléen du fait des or. 

Bref, à tester / rediscuter plus tard !